src/js/service/ImportService.js

```diff
@@ -148,8 +154,9 @@
    * @return {pskl.model.Piskel}
    * @private
    */
-  ns.ImportService.prototype.createPiskelFromImages_ = function (images,
+  ns.ImportService.prototype.createPiskelFromImages_ = function (images, name,
       frameSizeX, frameSizeY, smoothing) {
+    name = name || 'Imported piskel';
     var frames = this.createFramesFromImages_(images, frameSizeX, frameSizeY, smoothing);
     var layer = pskl.model.Layer.fromFrames('Layer 1', frames);
     var descriptor = new pskl.model.piskel.Descriptor('Imported piskel', '');
```

Comment:
Merge error here: Should pass `name` (which is given a default value above) instead of the hard-coded `'Imported piskel'` string here.